independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/BeanGenerator.java

```diff
@@ -249,6 +249,7 @@ Collection<Resource> generateSyntheticBean(BeanInfo bean) {
         implementGetKind(beanCreator, InjectableBean.Kind.SYNTHETIC);
         implementEquals(bean, beanCreator);
         implementHashCode(bean, beanCreator);
+        implementToString(beanCreator);
```

Comment:
Note that we can't implement the `Object#toString()` as a default method of `InjectableBean`...